torch/_functorch/aot_autograd.py

```diff
@@ -423,7 +420,7 @@
 @dynamo_timed
 def create_aot_dispatcher_function(
     flat_fn, flat_args: List[Any], aot_config: AOTConfig
-):
+) -> Tuple[Callable, ViewAndMutationMeta]:
```

Comment:
Looked into this a bit and it's actually a bit tricky, since we'd have to disentangle all the logic above when processing inputs. Because you'd want to share implementations and not copy paste that portion of the code, it almost forces this function to return weird tuples. But I think this is fixable, will attempt in my next diff